packages/suite-desktop-core/e2e/tests/passphrase/passphrase-cancel.test.ts

```diff
@@ -9,6 +9,8 @@ test.describe('Passphrase cancel', { tag: ['@group=passphrase'] }, () => {
     test('possible to cancel passphrase', async ({ page, devicePrompt, dashboardPage }) => {
         await dashboardPage.deviceSwitchingOpenButton.click();
         await dashboardPage.addHiddenWalletButton.click();
+        await page.getByTestId('@switch-device/add-existing-hidden-wallet-button').click();
```

Comment:
Ideally this should be also accessed thorugh `dashboardPage`, it's the new convention